bip-rpob-tx-template.mediawiki

```diff
@@ -0,0 +1,284 @@
+
+====Notes:====
+<references group="Note" />
+
+==Design==
+
+===Conceptual===
+''"Proof-of-Burn"'' transactions have long exited in Bitcoin, ''"OP_RETURN"'' transactions, (named after the op-code ''"return"'', that causes the public key script to return in failure, hence making the output unspendable and "burning" any value allocated to that output), these transactions have been actively mined since the release of Bitcoin-Core 0.9.
+
+Since OP_RETURN doesn't allow the public key script to be spent (as is it's design), there hasn't been any built-in solution addressing the requirements a application-useable proof-of-burn transaction, in particular:
+
+*A standard association with a public key in for transaction.
+*A standard protocol to publicly revoke a proof-of-burn transaction.
+
+In this proposal we satisfy these requirements by specifying a taproot public key in the op_return payload and an another output for revocation.
+
+===Other Requirements===
+Operation under <abbr title="Simple Payment Verification">SPV</abbr> assumptions:
+*Transactions are small: ''RPoB transactions (non-witness-part) are fixed to two possible sizes, depending if there is a change output, or not.''
+*Revocations are quickly and cheaply indexed: ''RPoB revocations only require obtaining the preimage to a double-sha256 hash. This is easily indexed and propagated in a gossip network.''
+
+Ease of implementation:
+*Use of only Taproot and Schnorr: ''To reduce the scope of the transaction, we use only Taproot for inputs and Outputs, and Schnorr for the signature that covers the funding outpoint.''
+
+Transaction funded by 3rd party:
+* It should be secure for an untrusted 3rd party fund the proof-of-burn transaction. ''Allowing for services to fund these transactions and be compensated with some sort of off-chain payment.''
+
+Future extendability:
+*Making use of the transaction version feature of bitcoin. ''This allows for easier indexing strategies in the future, (with a simple consensus change: to include in the coinbase the number of the transactions in the block by their transaction version).''
+* Including a byte for 8 version flag bits. ''This allows for a reliable way to upgrade in the future in a forward-comparable way.''
+
+===Transaction Design Overview:===
+[[File:bip-rpob-tx-template/rpob-template-summary-table.svg]]
+
+==Specification==
+This section specifies our RPoB transaction template.
+
+===Transaction===
+General form:
+<pre>
+version:      0x3
+marker:       0x0
+flag:         0x1
+txin_count:   0x1
+txout_count:  0x2 (without change) or 0x3 (with change)
+lock_time:    (any)
+</pre>
+
+*RPoB transactions MUST use the transaction version (nVersion) 0x03.
+
+*RPoB transactions MUST have only one input and either two or three outputs.
+
+===Input 1: Funding===
+General form:
+<pre>
+signature_script:  (empty)
+witness_script:    <taproot_witness>
+</pre>
+
+*The RPoB transaction input MUST be P2TR (Pay to Tap Root).
```

Comment:
Why?

Reply:
Again, to optimize for low complexity for SPV clients. Please the comment above.